### ISSUE TITLE
Improve the game

### ISSUE THREAD

**Issue by Vertfromage** [Vertfromage/ManFight#1]
This is a simple text based java game I made.  I thought it would be fun to let any other beginners who have been learning java contribute to it for Hacktoberfest... this will be my first year participating too.

Possible Improvements:
- Add documentation
- Add a monster class
- Add more fight moves 
- incorporate the random class
- Add insult method that throws out random insults and causes hurt feelings.

**Comment by Vertfromage** [comment]
Completed:
- Change the punch, kick, headbutt methods so that they can take a Man as input like: man1.punch(man2)  and injuries and response would all be controlled in that way.  This would also change how the ManFight class worked.   Thanks to RitikRawat

**Comment by ItsLaro** [comment]
Beginner here! I'd like to work on this.

**Comment by ivanreor** [comment]
I'd like to work on this!

**Comment by Vertfromage** [comment]
ItsLaro, you're welcome to join in.

**Comment by saguins** [comment]
wow. this sounds fun. I would like to add some new class, like magician guy. 🧙‍♂️

**Comment by Vertfromage** [comment]
Robert the Magician added to the Fight Club by saguins! Really fun code! Thanks saguins!

**Comment by RefreshingChi** [comment]
I think it would be interesting to add health to the man class. There should be ways to increase health at random, when you have a successful attack. Also if your health is over 50% your attack would be stronger. Any thoughts on adding something like this. I would like to do ti

**Comment by Vertfromage** [comment]
RefreshingChi, sounds interesting, how would you make health work with the injury of and amusing comments related to various body parts? Would each body part have it's own health or would injuries happen in relation to the health level, or randomly? Feel free to give it a go!

**Comment by Vertfromage** [comment]
SinSentido added a characterCreation class, Thanks SinSentido.

**Comment by wrongsyntax** [comment]
What sort of documentation is needed for this?

**Comment by Dovacody** [comment]
If it's okay, I Was wondering if I could add comments about what's happening to better document the code, and add todos so others can add documentation that I leave out. Thank you !

**Comment by RefreshingChi** [comment]
@Vertfromage I am thinking the overall character will start off with health of 100. 
If you are attacked with a punch you lose 10 health points, kick lose  20 health points, headbutt is 30 points and if the roar randomNumber equals 0 your health can increase 10. 

If you get your health to 125 it would regenerate one of your limbs(arm, leg) and your health would go to 75.

What you think of this

**Comment by Vertfromage** [comment]
wrongsyntax the documentation that could be added would be easy stuff like adding inline comments telling what methods do to make it easier for beginners to contribute.  I added a quick contributing.md, but it could be improved by someone who's managed an open source project before and knows what should be said in a contributing.md

Dovacody has the right idea, that's exactly what I was thinking of. 

I'm a beginner myself I opened up this project for contributions because I didn't find an easy enough java project to contribute to that wasn't just asking me to do code puzzles.  I figured it would be fun, but I don't have much experience with github, so just dealing with merge conflicts is a bit of a learning experience for me. :)
